android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -502,9 +507,9 @@ abstract class EngineSession(
             ),
 
             /**
-             * Combining the [RECOMMENDED] categories plus [SCRIPTS_AND_SUB_RESOURCES].
+             * Combining the [RECOMMENDED] categories plus [SCRIPTS_AND_SUB_RESOURCES] & getAntiTracking[EMAIL].
```

Comment:
@HarrisonOg thanks for updating the PR!
Just want to clarify if we need the `getAntiTracking[EMAIL]` in the comments instead of just & [EMAIL]? As I'm not sure if `getAntiTracking` is needed? Thanks in advance!